File: Code Generation/responses/cpp/deepseek/easy/find-the-largest-almost-missing-integer.cpp
#include <vector>
#include <unordered_map>
#include <algorithm>

using namespace std;

class Solution {
public:
    int largestInteger(vector<int> nums, int k) {
        int n = nums.size();
        unordered_map<int, int> freq;
        
        // Count frequency of each number in all subarrays of size k
        for (int i = 0; i <= n - k; i++) {
            unordered_map<int, bool> seen;
            for (int j = i; j < i + k; j++) {
                if (!seen[nums[j]]) {
                    freq[nums[j]]++;
                    seen[nums[j]] = true;
                }
            }
        }
        
        // Find the largest number that appears exactly once
        int ans = -1;
        for (auto& p : freq) {
            if (p.second == 1) {
                ans = max(ans, p.first);
            }
        }
        
        return ans;
    }
};

File: Code Generation/responses/python/deepseek/easy/find-the-largest-almost-missing-integer.py
from typing import List
from collections import defaultdict

class Solution:
    def largestInteger(self, nums: List[int], k: int) -> int:
        n = len(nums)
        freq = defaultdict(int)
        
        # Count frequency of each integer in all subarrays of size k
        for i in range(n - k + 1):
            subarray = nums[i:i + k]
            seen = set(subarray)  # Avoid counting duplicates in the same subarray
            for num in seen:
                freq[num] += 1
        
        # Find the largest integer that appears exactly once across all subarrays
        result = -1
        for num, count in freq.items():
            if count == 1:
                result = max(result, num)
        
        return result